message.go

```diff
@@ -234,6 +234,7 @@ type MessageSend struct {
 	AllowedMentions *MessageAllowedMentions `json:"allowed_mentions,omitempty"`
 	Reference       *MessageReference       `json:"message_reference,omitempty"`
 	StickerIDs      []string                `json:"sticker_ids"`
+	Flags           int                     `json:"flags,omitempty"`
```

Comment:
Feels to me like this should be the flag type that  describes the possible values in the other PR. I know there are places that do it like this as well but that seems like legacy @FedorLap2006 ?

Reply:
Fixed. Sorry for my rudimentary mistake.